spec/spec_helper.rb

```diff
@@ -1,13 +1,9 @@
 # This file is copied to spec/ when you run 'rails generate rspec:install'
-ENV["RAILS_ENV"] ||= "test"
+ENV["RAILS_ENV"] ||= 'test'
 require File.expand_path("../../config/environment", __FILE__)
-require "rspec/rails"
-require "capybara/email/rspec"
-require "sidekiq/testing/inline"
-require "vcr"
-
-Capybara.server_port = 52662
-Capybara.javascript_driver = :webkit
+require 'rspec/rails'
+require 'capybara/email/rspec'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.